### PR TITLE
Updated getting-started.md

### PR DESCRIPTION
**Pull request by TheDutchCoder**
* Made the diffs line up better
* Small spelling mistakes fixed
* Edited minor wording

### CODE REVIEW

File: content/guides/getting-started.md [hunk 1]
```diff
@@ -107,34 +107,34 @@ __src/index.js__
 ``` diff
 + import _ from 'lodash';
 +
-function component () {
-  var element = document.createElement('div');
+  function component () {
```
Comment (skipjack) [inline content/guides/getting-started.md]: `component () {` => `component() {`

**Comment by TheDutchCoder** [comment]
Done.

I always use `foo () {` for declarations and `foo()` for invocations, but it's a style preference 😉

**Comment by skipjack** [comment]
@TheDutchCoder definitely style but I think through the rest of the docs we're pretty consistent with no space. Thanks for updating!

**Comment by TheDutchCoder** [comment]
Makes sense! No problem!

**Comment by TheDutchCoder** [comment]
@skipjack could you check what's up with Travis? It doesn't seem to want to run :/

**Comment by skipjack** [comment]
Yeah not sure what's going on here but there may be some limit on how many builds we can do on Travis in a day or something. We've been hitting it pretty hard this morning. I'm going to cancel some of the builds and see if that helps. I think this may also have to do with #1326 which I'm not able to replicate but has been reported before when a lot of PRs are merged.

**Pull request merged**